src/main/java/seedu/address/logic/parser/FindAppointmentCommandParser.java

```diff
@@ -48,7 +50,7 @@ public FindAppointmentCommand parse(String args) throws ParseException {
     private Predicate<Appointment> parsePredicate(ArgumentMultimap argMultimap) throws ParseException {
         Predicate<Appointment> predicate = PREDICATE_SHOW_ALL_APPOINTMENTS;
 
-        if (!areAnyPrefixesPresent(argMultimap, PREFIX_NAME)) {
+        if (!areAnyPrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_DATETIME)) {
```

Comment:
Should we include in `MESSAGE_USAGE` for `FindAppointmentCommand` the parameters for `d/` too?
- Also should we let users know that they can use the name or date or both to find the appointment?

Reply:
Related to the `MESSAGE_USAGE` too, should we change it in `FindCommand` as well to include `Parameters: ENTITY_TYPE ENTITY_ARGUMENTS`, so that users know that they have to type in `find person` or `find appt` i.e. include the entity? (similar to `AddCommand`)

Reply:
Similarly for `FindPersonCommand`, can we include a `MESSAGE_USAGE` for it too?